src/bin/tikv-ctl.rs

```diff
@@ -1805,7 +1805,7 @@ fn convert_gbmb(mut bytes: u64) -> String {
     const GB: u64 = 1024 * 1024 * 1024;
     const MB: u64 = 1024 * 1024;
     if bytes < MB {
-        return bytes.to_string();
+        return format!("{} Byte ", bytes);
```

Comment:
How about write the logic in `convert_gbmb`?

Reply:
I think `{} B` is enough. see https://en.wikipedia.org/wiki/File_size

Reply:
@siddontang OK, I changed unit Byte to B.